### ISSUE TITLE
Should it colorize .t4 files also?

### ISSUE THREAD

**Issue by rpokrovskij** [olegsych/T4Toolbox#16]
Hi I have found that T4toobox doesn't colorize templates saved as .t4 files (that I use in MVC project for scaffolding ).

**Comment by olegsych** [comment]
That's a good idea, @rpokrovskij. :+1: Thanks for submitting it.

I think this only requires exporting an additional `FileExtensionToContentTypeDefinition` to associate `.t4` extension with the `TextTemplate` content type in the [TemplateContentType.cs](https://github.com/olegsych/T4Toolbox/blob/master/src/T4Toolbox.VisualStudio.Editor/TemplateContentType.cs).

**Comment by ghost** [comment]
Agree: Good idea.

**Comment by tom-englert** [comment]
Yes, id definitely should! Any ideas when this will be added? Or are you waiting for someone to provide a PR?

**Comment by ghost** [comment]
Adding my 👍 for this feature.  Having a T4 script that takes 5-15 seconds to run means that you probably want to run it on demand, which means switching to a `.tt` / `.t4` include-file approach to managing code.  And without color coding available for the `.t4` file, you're basically coding in Notepad again, which is no fun.
